src/store/index.ts

```diff
@@ -3,9 +3,9 @@ import userReducer from "./reducer/userSlice";
 import tileReducer from "./reducer/tileSlice";
 import courtColorReducer from "./reducer/courtColorSlice";
 import rulerControlReducer from "./reducer/rulerControlSlice";
-import { courtsApi } from "../redux/api/courtSizeApi";
-import { priceApi } from "../redux/api/priceApi";
-import { designApi } from "../redux/api/designApi";
+import { courtsApi } from "@/redux/api/courtSizeApi";
+import { priceApi } from "@/redux/api/priceApi";
+import { designApi } from "@/redux/api/designApi";
```

Comment:
Did you add cartApi to the store? Got an error message: "Error: No data found at `state.shoppingCart`. Did you forget to add the reducer to the store?"